src/sentry/static/sentry/app/views/onboarding/integrationSetup.tsx

```diff
@@ -0,0 +1,305 @@
+            organization={organization}
+            priority="primary"
+            size="small"
+          />
+        </DocsWrapper>
+      </React.Fragment>
+    );
+  }
+
+  renderPostInstallText() {
+    const {provider} = this.state;
+    if (!provider) {
+      return null;
+    }
+    //TODO: dyanically determine the snippet based on the language
+    return (
+      <div>
+        <p>
+          {t(
+            "Congrats, you just installed the %s integration! Now that it's is installed, the next time you trigger an error it will go to your Sentry.",
+            provider.name
+          )}
+        </p>
+        <p>
+          {t(
+            'This snippet includes an intentional error, so you can test that everything is working as soon as you set it up:'
+          )}
+        </p>
+        <div>
+          <CodeWrapper>
+            <code>
+              <TokenFunction>myUndefinedFunction</TokenFunction>
+              <TokenPunctuation>();</TokenPunctuation>
+            </code>
+          </CodeWrapper>
+        </div>
+        <p>
+          {t(
+            "If you're new to Sentry, use the email alert to access your account and complete a product tour."
+          )}
+        </p>
+        <p>
+          {t(
+            "If you're an existing user and have disabled alerts, you won't receive this email."
+          )}
+        </p>
+      </div>
+    );
+  }
+
+  renderPostInstallInstructions() {
+    const {organization, project} = this.props;
+    const {provider} = this.state;
+    if (!project || !provider) {
+      return null;
+    }
+    return (
+      <React.Fragment>
+        {this.renderSetupInstructions()}
+        {this.renderPostInstallText()}
+        <FirstEventFooter
+          project={project}
+          organization={organization}
+          docsLink={this.platformDocs}
+          docsOnClick={this.handleFullDocsClick}
+        />
+      </React.Fragment>
+    );
+  }
+
+  render() {
+    const {platform} = this.props;
+    const {hasError} = this.state;
+
+    const loadingError = (
+      <LoadingError
+        message={t('Failed to load the integration for the %s platform.', platform)}
+        onRetry={this.fetchData}
+      />
+    );
+
+    const testOnlyAlert = (
+      <Alert type="warning">
+        Platform documentation is not rendered in for tests in CI
+      </Alert>
+    );
+
+    return (
+      <React.Fragment>
+        {this.state.installed
+          ? this.renderPostInstallInstructions()
+          : this.renderIntegrationInstructions()}
+        {getDynamicText({
+          value: !hasError ? null : loadingError,
+          fixed: testOnlyAlert,
+        })}
+      </React.Fragment>
+    );
+  }
+}
+const CodeWrapper = styled('pre')`
+  padding: 1em;
+  overflow: auto;
+  background: #251f3d;
+  font-size: 15px;
+`;
+
+const TokenFunction = styled('span')`
+  color: #7cc5c4;
+`;
+
+const TokenPunctuation = styled('span')`
+  color: #b3acc1;
+`;
```

Comment:
lol this is ghetto af but its fine

Reply:
The alternative was using Gatsby classes lol

Reply:
Tbh that might be better since the color is only defined in one place then. yeah I might actually switch it